test/test.jwt.ts

```diff
@@ -643,7 +612,7 @@ describe('.fromAPIKey', () => {
     it('should set the .apiKey property on the instance', (done) => {
       jwt.fromAPIKey(KEY, (err) => {
         assert.strictEqual(jwt.apiKey, KEY);
-        assert.strictEqual(err, null);
+        assert.equal(err, null);
```

Comment:
why was this change needed?

Reply:
In the `fromAPIKey` callback I had it set up so that the callback could just be invoked with no parameters, making `err` undefined instead of null.  Not sure what the best practice is here, or if it matters.  I could make err `err: Error|null` instead of `err?: Error` if you think that's better :)